ChiModules/KEigenvalueSolver/IterativeMethods/power_iteration.cc

```diff
@@ -156,8 +158,8 @@ void KEigenvalue::Solver::PowerIteration(LBSGroupset& groupset)
     chi_log.ProcessEvent(source_event_tag,
                          ChiLog::EventOperation::AVERAGE_DURATION);
   size_t num_angles = groupset.quadrature->abscissae.size();
-  long int num_unknowns = (long int)glob_dof_count*
-                          (long int)num_angles*
+  long int num_unknowns = (long int)glob_node_count *
```

Comment:
is this an example where C++ casting should be used?

Reply:
Yes. This code was changed as part of a refactor but I went ahead and changed the obvious c-style casts in the file. Also cleaned it up a bit on some places where clang-tidy complained.